src/todoapp/components/TodoApp.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+    <h1>Список завдань</h1>
+    <TodoAppEdit></TodoAppEdit>
+    <hr />
+    <TodoAppItem v-for="task in tasks" :key="task.id" :task="task"></TodoAppItem>
+  </div>
+</template>
+
+<script lang="ts">
+import TodoAppEdit from '@/todoapp/components/TodoAppEdit.vue';
+import TodoAppItem from './TodoAppItem.vue';
+
+import { defineComponent, computed } from 'vue';
+
+import { useStore } from 'vuex';
+import { StoreModuleEnum } from '@/store/types';
+import { TodoAppGettersEnum, TodoAppActionEnum } from '@/todoapp/store//types';
+export default defineComponent({
```

Comment:
It's better to import core dependencies and then custom components. Also separate import and export section with an empty line.